incubator/keycloak/templates/statefulset.yaml

```diff
@@ -84,14 +84,14 @@ spec:
             httpGet:
               path: /auth/
               port: http
-            initialDelaySeconds: 120
-            timeoutSeconds: 5
+            initialDelaySeconds: {{ default 120 .Values.keycloak.livenessInitialDelaySeconds }}
```

Comment:
Remove default function calls. They are redundant. Defaults should be in `values.yaml`.